addon/mixins/dom.js

```diff
@@ -110,13 +110,13 @@ export default Mixin.create({
    ```
 
    @method addEventListener
-   @param { String } selector the jQuery selector or element
+   @param { String } selector the DOM selector or element
    @param { String } _eventName the event name to listen for
    @param { Function } _callback the callback to run for that event
    @public
    */
   addEventListener(selector, eventName, callback, _options) {
-    assert('Must provide an element (not a jQuery selector) when using addEventListener in a tagless component.', this.tagName !== '' || typeof selector !== 'string');
+    assert('Must provide an element (not a DOM selector) when using addEventListener in a tagless component.', this.tagName !== '' || typeof selector !== 'string');
     assert('Called addEventListener before the component was rendered', this._currentState === this._states.inDOM);
 
     // Ember.assign would be better here, but Ember < 2.5 doesn't have that :(
```

Comment:
Since we are bumping major versions, and dropping Ember 2.4 support we should be able to use `Ember.assign` and kill this comment (and the `merge(first, merge(second, third))` BS).